rpcs3/Emu/Cell/SPULLVMRecompiler.cpp

```diff
@@ -4813,6 +4822,38 @@ class spu_llvm_recompiler : public spu_recompiler_base, public cpu_translator
 	void ROTQBI(spu_opcode_t op)
 	{
 		const auto a = get_vr(op.ra);
+		const auto ax = get_vr<u8[16]>(op.ra);
+		const auto bx = get_vr<u8[16]>(op.rb);
+
+		// Combined bit and bytes shift
+		if (auto [ok, v0, v1] = match_expr(ax, rotqbybi(match<u8[16]>(), match<u8[16]>())); ok && v1.eq(bx))
+		{
+			const auto b32 = get_vr<s32[4]>(op.rb);
+			// Is the rotate less than 31 bits?
+			if (auto k = get_known_bits(b32); !!(k.Zero & 0x60))
```

Comment:
It was checking for any of them being 0 which is wrong.
```suggestion
			if (auto k = get_known_bits(b32); (~k.Zero & 0x60) == 0)
```